assets/js/bootstrap-datepicker.js

```diff
@@ -0,0 +1,2096 @@
+/*!
```

Comment:
the minified version shows which version it is based on, so I don't think we need the full-version here for reference.